mitiq/zne/scaling/tests/test_folding.py

```diff
@@ -931,6 +618,14 @@ def test_fold_left_and_fold_right_match_on_odd_scale_factors():
             require_measurement_equality=True,
         )
 
+    for s in (3, 7, 15):
+        assert _equal(
+            fold_gates_from_left(circuit, s),
+            fold_gates_at_random(circuit, s),
```

Comment:
Why don't you test `from_right` here too?

Reply:
I think it is redundant. 
First we test that `from_left == from_right` (right above),  then we test `from_left == at_random` .  So if both tests pass, it should automatically imply that `from_right == at_random` .

Reply:
Oh, missed it above because it didn't show up in the diff, oops!